assets/training/general/environments/acpt-pytorch-2.0-cuda11.7/context/requirements.txt

```diff
@@ -12,4 +12,6 @@ azureml-inference-server-http~=0.8.0
 inference-schema~=1.5.0
 MarkupSafe==2.1.2
 regex
-pybind11
\ No newline at end of file
+pybind11
+cryptography>=40.0.2
+aiohttp>=3.8.5
```

Comment:
you might want to add this fix to other acpt envs also

Reply:
Yes, will do this as separate PRs, for both operational and in case it breaks reason